{
    "command.error.internal": "No se pudo procesar este comando. {error}"
}